### PR TITLE
test utils: lookupCNAME explained and safe

### PR DESCRIPTION
**Pull request by alebedev87**

### CODE REVIEW

File: test/e2e/util.go [hunk 1]
```diff
@@ -214,20 +214,34 @@ func rootCredentials(kubeClient client.Client, name string) (map[string][]byte,
...
+	if host[len(host)-1] != '.' {
+		host += "."
+	}
+	m.SetQuestion(host, dns.TypeCNAME)
 	r, _, err := c.Exchange(&m, server+":53")
 	if err != nil {
-		return nil, err
+		return "", err
+	}
+	if len(r.Answer) == 0 {
```
Comment (arjunrn) [inline test/e2e/util.go:237]: Isn't this an error in our case?

Comment (alebedev87) [reply test/e2e/util.go:237]: Yes, I think it's better to an error case.

**Comment by alebedev87** [comment]
`Error fetching blob: invalid status code from registry 500 (Internal Server Error)`

**Pull request merged**